oaknational/Oak-Web-Application
# feat: skip to main content button - LESQ-167

## Description



- Adds 'Skip to content' button link to layout component. 
- This is only visible when focused and disappears on blur.
- Will take focus to main when clicked

## Issue(s)



## How to test

1. Go to https://deploy-preview-2592--oak-web-application.netlify.thenational.academy
Load any teacher page, press tab, you should 'Skip to content' button appear on focus. 
When clicked it will take your focus 'main' and disappear. 

## Checklist
